embeddings/evaluator/metrics_evaluator.py

```diff
@@ -1,22 +1,22 @@
 import abc
-from typing import Any, Dict, List, Tuple
+from typing import Any, Dict, List, Sequence, Union
 
 import numpy as np
-from datasets import Metric
+import torch
 
 from embeddings.evaluator.evaluator import Evaluator
+from embeddings.metric.metric import Metric
 
 
 class MetricsEvaluator(Evaluator[Dict[str, np.ndarray], Dict[str, Any]]):
     @property
     @abc.abstractmethod
-    def metrics(self) -> List[Tuple[Metric, Dict[str, Any]]]:
+    def metrics(
+        self,
+    ) -> Sequence[Metric[Union[List[Any], np.ndarray, torch.Tensor], Dict[Any, Any]]]:
         pass
 
     def evaluate(self, data: Dict[str, np.ndarray]) -> Dict[str, Any]:
         return {
-            metric.name: metric.compute(
-                references=data["y_true"], predictions=data["y_pred"], **kwargs
-            )
-            for metric, kwargs in self.metrics
+            str(metric): metric.compute(data["y_true"], data["y_pred"]) for metric in self.metrics
```

Comment:
I would use keyword args like previously  `y_true=data["true"]` etc.

Reply:
Kamil asked to pass args by key, rather than changing `y_true` to `true` (typo)